spec/swagger_helper.rb

```diff
@@ -159,7 +159,7 @@
 end
 
 def encoded_header
-  Base64.encode64(x_rh_identity.to_json)
+  Base64.strict_encode64(x_rh_identity.to_json)
```

Comment:
@akofink This was missing - `strict` removes linebreaks which was apparently causing problems with Swagger specs